modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java

```diff
@@ -795,35 +802,42 @@ private String _getNavigationItemHref(
 
 		List<DDMFormField> nestedDDMFormFields = new ArrayList<>();
 
-		for (ObjectLayoutRow objectLayoutRow :
-				objectLayoutBox.getObjectLayoutRows()) {
+		try {
```

Comment:
Hey @mateussgn ,
I believe IF somehow the code goes wrong (go to the catch), the iteration will be interrupted because the try/catch is outside the iteration. Is this effect that you planned?

Reply:
You're right @marcelabc. Thanks for the tip!